src/dto/mllib.hpp

```diff
@@ -176,6 +176,15 @@ namespace dd
       };
       DTO_FIELD(UnorderedFields<Any>, template_params);
 
+      DTO_FIELD_INFO(softmax_postprocess)
+      {
+        info->description
+            = "Post process the model output by applying a softmax. If a "
+              "model "
+              "already applies a sigmoid, and this should be set to false";
+      }
+      DTO_FIELD(Boolean, softmax_postprocess) = true;
```

Comment:
We might want a higher level name for this parameter. In your case it's multilabel, but maybe we don't need the softmax in other cases either. @beniz any opinion on this?

Reply:
In caffe we had the `multi_label` option, which effectively disabled the softmax: https://github.com/jolibrain/deepdetect/blob/master/src/backends/caffe/caffeinputconns.h#L193. It was in the input connector, but I think you could put it in mllib, since this is also arguably a property of the model/of the output.

Note that the softmax layer is already not used in some cases (eg regression), in general we prefer to let the user chose the type of tasks they want to perform, and do all the dirty work of plugging layers together and post processing the output behind the scenes